src/coinbase/tests/e2e.ts

```diff
@@ -0,0 +1,102 @@
+      privateKey: process.env.PRIVATEKEY,
+    });
+  });
+  it("should be able to access environment variables", () => {
+    expect(process.env.NAME).toBeDefined();
+    expect(process.env.PRIVATEKEY).toBeDefined();
+  });
+  it("should be able to interact with the Coinbase SDK", async () => {
+    console.log("Fetching default user...");
+    const user = await coinbase.getDefaultUser();
+    expect(user.getId()).toBeDefined();
+    console.log(`Fetched default user with ID: ${user.getId()}`);
+
+    console.log("Creating new wallet...");
+    const wallet = await user.createWallet();
+    expect(wallet?.getId()).toBeDefined();
+    console.log(
+      `Created new wallet with ID: ${wallet.getId()}, default address: ${wallet.getDefaultAddress()}`,
+    );
+
+    console.log("Importing wallet with balance...");
+    const seedFile = JSON.parse(process.env.SEED || "");
+    const walletId = Object.keys(seedFile)[0];
+    const seed = seedFile[walletId].seed;
+
+    const userWallet = await user.importWallet({ seed, walletId });
+    expect(userWallet).toBeDefined();
+    expect(userWallet.getId()).toBe(walletId);
+    console.log(
+      `Imported wallet with ID: ${userWallet.getId()}, default address: ${userWallet.getDefaultAddress()}`,
+    );
+    await userWallet.saveSeed("test_seed.json");
+
+    try {
+      await userWallet.faucet();
+    } catch {
+      console.log("Faucet request failed. Skipping...");
+    }
+    console.log("Listing wallet addresses...");
+    const addresses = userWallet.listAddresses();
+    expect(addresses.length).toBeGreaterThan(0);
+    console.log(`Listed addresses: ${userWallet.listAddresses().join(", ")}`);
+
+    console.log("Fetching wallet balances...");
+    const balances = await userWallet.listBalances();
+    expect(Array.from([...balances.keys()]).length).toBeGreaterThan(0);
+    console.log(`Fetched balances: ${balances.toString()}`);
+
+    console.log("Exporting wallet...");
+    const exportedWallet = await wallet.export();
+    expect(exportedWallet.walletId).toBeDefined();
+    expect(exportedWallet.seed).toBeDefined();
+
+    console.log("Saving seed to file...");
+    await wallet.saveSeed("test_seed.json");
+    expect(fs.existsSync("test_seed.json")).toBe(true);
+    console.log("Saved seed to test_seed.json");
+
+    const unhydratedWallet = await user.getWallet(walletId);
+    expect(unhydratedWallet.canSign()).toBe(false);
+    await unhydratedWallet.loadSeed("test_seed.json");
+    expect(unhydratedWallet.canSign()).toBe(true);
+    expect(unhydratedWallet.getId()).toBe(walletId);
+
+    console.log("Transfering 1 Gwei from default address to second address...");
+    const transfer = await unhydratedWallet.createTransfer(1, Coinbase.assets.Gwei, wallet);
+    expect(await transfer.getStatus()).toBe(TransferStatus.COMPLETE);
+    console.log(`Transferred 1 Gwei from ${unhydratedWallet} to ${wallet}`);
+
+    console.log("Fetching updated balances...");
```

Comment:
can we add one more transfer request that transfers funds back to the CB faucet `0x6Cd01c0F55ce9E0Bf78f5E90f72b4345b16d515d` ?

Reply:
Synced on this but the faucet contract is updated. we will have a followup